packages/flutter_tools/templates/create/ios.tmpl/Runner/PluginRegistry.h.tmpl

```diff
@@ -0,0 +1,24 @@
+//
+//  Generated file. Do not edit.
+//
+
+#ifndef PluginRegistry_h
+#define PluginRegistry_h
+
+#import <Flutter/Flutter.h>
+
+{{#withPluginHook}}
+#import "{{pluginClass}}.h"
+{{/withPluginHook}}
+
+@interface PluginRegistry : NSObject
+
+{{#withPluginHook}}
+@property (readonly, nonatomic) {{pluginClass}}* {{pluginProjectName}};
```

Comment:
Apple seems to prefer {{pluginClass}} *{{pluginProjectName}} and we should do that for consistency with other templates.